extensions/podman/src/compatibility-mode.spec.ts

```diff
@@ -240,12 +240,12 @@ test('darwin: test promptRestart IS NOT ran when findRunningMachine returns unde
   vi.mock('./extension', () => {
     return {
       findRunningMachine: () => {
-        return Promise.resolve(undefined);
+        return Promise.resolve();
```

Comment:
we need to test that findRunningMachine returns undefined as the function returns undefined, if there is nothing available.